spec/models/mixins/ansible_playbook_mixin_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe AnsiblePlaybookMixin do
+  let(:test_instance) do
+    Class.new(ActiveRecord::Base) do
+      def self.name; "TestClass"; end
+      self.table_name = "services"
+      include AnsiblePlaybookMixin
+    end.new
+  end
+
+  let(:job)    { FactoryBot.create(:embedded_ansible_job, :name => 'ansible_playbook_mixin_test') }
+  let(:task)   { FactoryBot.create(:miq_task) }
+  let(:status) { ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job::Status.new(task, MiqTask::STATUS_OK) }
+
+  context "#playbook_log_stdout" do
```

Comment:
@djberg96 Since we are adding specs for pay playbook_log_stdout, perhaps we should add a guard clause for nil and raise an ArgumentError (and then add a spec for that case?)  This way, at least, if a future caller calls with nil again at least we know why it's failing.

Reply:
Do you want that in this PR or a separate one since we're now not actually touching that method?

Reply:
I'm fine with it in this PR.

Reply:
@Fryguy ok, added. Keep in mind that `ArgumentError` is a subclass of `StandardError`, so it was getting caught by the current `rescue` clause, so the log message needed extra handling.